ehitchhiking-ui/src/styles/constants.sass

```diff
@@ -3,3 +3,4 @@ $default-padding: 5%
 $trip-card-height: 4.5rem
 $height-empty-block: 10px
 $number-of-trips-in-list: 4
+$height-borders-of-trip-in-list: 2px
```

Comment:
There is no such dimension for border as height. Pls rename collection-item-border-width

Reply:
fixed